### PR TITLE
Make blog posts share- and SEO-friendly

### PR DESCRIPTION
**Pull request by Proghat**
https://trello.com/c/mpxS8Uby/194-when-i-share-a-post-in-social-network-i-want-to-see-preview-of-post-so-my-friends-will-see-what-the-post-about

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://meta-tags.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://meta-tags.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: config/initializers/meta_tags.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+MetaTags.configure do |c|
+  c.title_limit        = 100
```
Comment (ezhlobo) [inline config/initializers/meta_tags.rb]: Maximum 75-80

File: config/initializers/meta_tags.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  c.description_limit  = 160
```
Comment (ezhlobo) [inline config/initializers/meta_tags.rb]: 140

File: config/initializers/meta_tags.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  c.keywords_limit     = 255
```
Comment (ezhlobo) [inline config/initializers/meta_tags.rb]: 250

File: config/initializers/meta_tags.rb (hunk 2 above)
Comment (Proghat) [inline config/initializers/meta_tags.rb]: Why so? 160 is default.

File: config/initializers/meta_tags.rb (hunk 1 above)
Comment (Proghat) [inline config/initializers/meta_tags.rb]: It limits length of `<title>` tag. So I'd like to keep in mind that we should keep length of title below 80 chars but meta-tags should never truncate it.

File: config/initializers/meta_tags.rb (hunk 3 above)
Comment (Proghat) [inline config/initializers/meta_tags.rb]: Is it big difference?

File: config/initializers/meta_tags.rb (hunk 2 above)
Comment (ezhlobo) [inline config/initializers/meta_tags.rb]: @Proghat I was confused. it doesn't make sense for google and yandex now. And keywords too.

File: config/initializers/meta_tags.rb (hunk 1 above)
Comment (ezhlobo) [inline config/initializers/meta_tags.rb]: @Proghat maybe we shouldn't use `meta-tags` for title? But I haven't found any rules about length of title.

Keeping in mind is bad practice BTW.

Comment (Proghat) [inline config/initializers/meta_tags.rb]: > Keeping in mind is bad practice BTW.

I agree. What do you thing about input in control panel which will be colored in yellow when title is more than 80 chars and in red if title is more than 100 chars?

> maybe we shouldn't use meta-tags for title?
Let's leave it for now.

File: config/initializers/meta_tags.rb (hunk 2 above)
Comment (Proghat) [inline config/initializers/meta_tags.rb]: Good :+1:

**Pull request merged**